files/mcollective/pluginpackager/aiomodulepackage_packager.rb

```diff
@@ -183,11 +194,13 @@ def validate_environment
       end
 
       def assert_new_enough_puppet
-        unless File.executable?("/opt/puppetlabs/bin/puppet")
-          raise("Cannot build package. '/opt/puppetlabs/bin/puppet' is not present on the system.")
+        puppet_bin = which('puppet')
+        puppet_bin = '/opt/puppetlabs/bin/puppet' if !puppet_bin
```

Comment:
thanks, yeah this will help.

Sorry to be a total drag but would you mind changing all the `'` to `"`?